### PR TITLE
Add kernel_vmstat input plugins

### PR DESCRIPTION
**Pull request by robinpercy**
Input that gathers statistics from `/proc/vmstat`

### CODE REVIEW

**Comment by sparrc** [comment]
looks good but same revisions are needed as what I said in #1164

**Comment by robinpercy-xm** [comment]
Revisions have been made. I didn't add a "notlinux" dummy file to this one, since it's part of the system package. Let me know if that's wrong.

File: plugins/inputs/system/kernel_vmstat.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+type KernelVmstat struct {
+	statFile string
+}
+
+func (k *KernelVmstat) Description() string {
+	return "Get kernel statistics from /proc/vmstat"
+}
+
+func (k *KernelVmstat) SampleConfig() string {
+	return `[[inputs.kernel_vmstat]]`
```
Comment (sparrc) [inline plugins/inputs/system/kernel_vmstat.go]: this should be `return ""`

**Comment by sparrc** [comment]
sorry for the hassle but please rebase this and #1164 and then I can merge

**Comment by robinpercy-xm** [comment]
No worries. Glad to see so much activity here. Both have been rebased.

**Pull request closed without merging**